test/autests/gold_tests/body_verification/body_verification.test.py

```diff
@@ -0,0 +1,77 @@
+'''
+Verify basic body verification functionality.
+'''
+# @file
+#
+# Copyright 2022, Verizon Media
+# SPDX-License-Identifier: Apache-2.0
+#
+
+Test.Summary = '''
+Verify basic body verification functionality.
+'''
+
+#
+# Test 1: Verify bodies can be verified correctly for HTTP/1.1.
+#
+r = Test.AddTestRun("Verify bodies can be verified correctly for HTTP/1.1.")
+client = r.AddClientProcess("client1", "http.yaml")
+server = r.AddServerProcess("server1", "http.yaml")
```

Comment:
It looks like we are testing the `equals` and `contains` directives in these tests (unless I missed others). We do not need to add a test for every directive, but it would be good to also test the no-case and negation/`not` functionality just to verify those got plumbed correctly as well.